tests/e2e/specs/out-of-game/localization.spec.js

```diff
@@ -72,3 +72,50 @@ describe('Localization', () => {
     checkLobbyTranslation(fr);
   });
 });
+
+describe('language files', () => {
+  function extractKeys(obj, keyList = []) {
+    Object.keys(obj).forEach(key => {
+      if (typeof obj[key] === 'object' && obj[key] !== null) {
+        extractKeys(obj[key], keyList);
+      } else {
+        keyList.push(key);
+      }
+    });
+    return keyList;
+  }
+
+  let enKeys = extractKeys(en).sort();
+  let frKeys = extractKeys(fr).sort();
+  let esKeys = extractKeys(es).sort();
+
+  it('English should have no empty strings', () => {
+    enKeys.forEach((key) => {
+      expect(en[key]).to.not.eql('');
+    });
+  });
+
+  it('French should have the same keys', () => {
+    for(let i = 0; i < enKeys.length; i++) {
+      expect(enKeys[i]).to.eql(frKeys[i]);
+    }
```

Comment:
Same idea as above; let's use a descriptive error message so it's easier to identify which key failed, why
```suggestion
    for (let i = 0; i < enKeys.length; i++) {
      const assertionMsg = `fr.json should have the key ${ enKeys[i] } for key number ${ i }, but instead it had key ${ frKeys[i] }`;
      expect(enKeys[i]).to.eq(frKeys[i], assertionMsg);
    }
```